### DISCUSSION TITLE
v5.0.0-alpha.1

### DISCUSSION THREAD

**Discussion by joschi** [dropwizard/dropwizard#8097] [Announcements]
## What's Changed
* Upgrade to Jakarta EE 10 components by @joschi in https://github.com/dropwizard/dropwizard/pull/8038
* fix(deps): update dependency org.glassfish.jersey:jersey-bom to v3.1.5 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8042
* fix(deps): update dependency org.hibernate.orm:hibernate-core to v6.2.15.final (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8043
* fix(deps): update dependency jakarta.xml.bind:jakarta.xml.bind-api to v4.0.1 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8041
* chore(deps): update github/codeql-action digest to 03e7845 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8040
* fix(deps): update jetty.version to v12.0.4 (release/5.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/8044
* chore(deps): update github/codeql-action action to v3 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8049
* chore(deps): update actions/upload-artifact action to v4 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8048
* Fix CodeQL workflow by @joschi in https://github.com/dropwizard/dropwizard/pull/8050
* fix(deps): update dependency org.hibernate.orm:hibernate-core to v6.4.1.final (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8046
* fix(deps): update dependency org.checkerframework:checker-qual to v3.42.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8057
* Add GracefulHandler for a graceful shutdown by @zUniQueX in https://github.com/dropwizard/dropwizard/pull/8060
* fix(deps): update dependency org.javassist:javassist to v3.30.1-ga (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8070
* chore(deps): update dependency org.apache.maven.plugins:maven-compiler-plugin to v3.12.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8084
* chore(deps): update dependency org.apache.maven.plugins:maven-compiler-plugin to v3.12.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8085
* fix(deps): update dependency org.liquibase:liquibase-core to v4.25.1 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8086
* fix(deps): update dependency com.google.guava:guava to v33 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/8087
* fix(deps): update jetty.version to v12.0.5 (release/5.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/8092

**Full Changelog**: https://github.com/dropwizard/dropwizard/compare/v4.0.5...v5.0.0-alpha.1

<hr /><em>This discussion was created from the release <a href='https://github.com/dropwizard/dropwizard/releases/tag/v5.0.0-alpha.1'>v5.0.0-alpha.1</a>.</em>

**Comment by 474846718** [comment]
Do we have any plan to adopt the MicroProfile standard for Dropwizard's future version?
The latest MicroProfile version currently includes the Jakarta EE 10 core.
I think now it's a good time to do it in the 5.0.0 version if we have a plan.

**Reply by joschi** [reply]
@474846718 What advantages would you see in supporting [MicroProfile](https://microprofile.io/) given that there are already various different frameworks implementing it?
